app/src/main/res/values-cs/strings.xml

```diff
@@ -0,0 +1,27 @@
+<resources>
+    <string name="app_name">Storyous Assistant</string>
+    <string name="no_permissions_camera">Aplikace potřebuje použít fotoaparát k jednorázovému načtení QR kódu z pokladny.</string>
+    <string name="give_camera_permission">Povolit přístup ke fotoaparátu</string>
+    <string name="remove_configuration">Odpárovat pokladnu</string>
+    <string name="synchronize">Propisovat telefonní čísla do poklady</string>
+    <string name="notification_info">Vaše příchozí hovory se nyní synchronizují do pokladního systému Storyous.</string>
+    <string name="config_error">Nastala chyba při konfiguraci. Zkuste to prosím později.</string>
+
+    <string-array name="config_instructions">
+        <item>Spárování aplikace s pokladním systémem Storyous:</item>
+        <item>V pokladním systému Storyous otevřete menu pomoct tlačítka v levém horním rohu.</item>
```

Comment:
otevřete menu pomoc**í** tlačítka

Reply:
jj, tohle je překlep, to by chtělo určitě opravit :)